desktop/core/src/desktop/js/apps/jobBrowser/app.js

```diff
@@ -14,15 +14,111 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import $ from 'jquery';
+import * as ko from 'knockout';
+
 import huePubSub from 'utils/huePubSub';
+
 import './components/hiveQueryPlan/webcomp';
 import './components/queriesList/webcomp';
 import './components/impalaQueries/webcomp';
+import './eventListeners';
+import JobBrowserViewModel from './knockout/JobBrowserViewModel';
+import Job from './knockout/Job';
+import I18n from 'utils/i18n';
 
 huePubSub.subscribe('app.dom.loaded', app => {
   if (app !== 'jobbrowser') {
     return;
   }
 
-  // TODO: Move js from job_browser.mako here.
+  $(document).ready(() => {
```

Comment:
So this is only called once, which is why we don't need to cleanup subscriptions?

Reply:
Correct.